### ISSUE TITLE
Where is the manual?

### ISSUE THREAD

**Issue by lucmp** [flareteam/flare-game#813]
I have a slingshot but for the life of me I can't figure out how to use it. What do I need the clothes for? How do I get promoted to level 2, 3 etc.? Can I climb up those things that look like ladders on the walls of the zombie cave?

I've searched high and low and can't find it anywhere. And I can't really play the game. Is there one? Is there going to be one?

**Comment by McSinyx** [comment]
I agree, there indeed should be some kind of in-game hint sometimes. But really, did you skip the intro?

> I have a slingshot but for the life of me I can't figure out how to use it.

At the beginning, the game tells you to use `${INPUT_ATTACK}` (usually left mouse) to use the primary attack.

**Comment by dorkster** [comment]
I'm also for a more traditional manual outside of in-game tutorials. We kind of make the assumption that the player has experience playing other ARPGs, or other video games at the least. A manual could cover a few things:

- the basics of controlling the game, including navigating menus
- introduce the game play mechanics; stats/leveling, equipment, powers, etc
- a brief strategy guide for the beginning of the Empyrean campaign (I always liked manuals for old games that did this)
- a description of the configuration menu options
- possibly introduce how to mod the game? Show how to change the behavior of an item or power
